core/src/main/scala/org/http4s/rho/bits/ResponseGenerator.scala

```diff
@@ -56,7 +56,7 @@ abstract class EntityResponseGenerator(val status: Status) extends ResponseGener
   def apply[A](body: A, headers: Headers)(implicit w: EntityEncoder[A]): Task[T[A]] = {
     w.toEntity(body).flatMap { case Entity(proc, len) =>
       val hs = len match {
-        case Some(l) => (w.headers ++ headers).put(`Content-Length`(l))
+        case Some(l) => (w.headers ++ headers).put(`Content-Length`.unsafeFromLong(l))
```

Comment:
I'm okay with these, because an Entity should never return a negative length, and if it does, we'll get a failed task.

(It does bring up that we can unsafely construct `Entities`, but that's a separate issue.  And once again, I wish we had a native unsigned long.)